pyproject.toml

```diff
@@ -6,7 +6,7 @@ requires = [
 ]
 
 [project]
-name = "em-keyboard"
+name = "em"
```

Comment:
I believe this we can't change as it'd try to change distribution name. [em](https://pypi.org/project/em/) seems reserved on PyPI.

It seems though that the project is [archived](https://github.com/ikalnytskyi/em), probably abandoned, so it may be possible to request removal for use in this project.